robot_modeling/DQ_SerialManipulatorDH.m

```diff
@@ -50,10 +52,17 @@
 %
 % Contributors to this file:
 %     Murilo M. Marinho - murilo@nml.t.u-tokyo.ac.jp
+%     Juan Jose Quiroz Omana - juanjqo@g.ecc.u-tokyo.ac.jp
```

Comment:
@bvadorno that was my fault. In the beginning, I assumed that since I was creating a new m-file I would be the only contributor, even if the code of the new file was based on another implementation.  I made the same error when I implemented the class DQ_SerialManipulatorMDH in C++ and @mmmarinho pointed out the same question. After a brief internal discussion, I understood my error. I made the respective corrections in C++. However, the respective correction in Matlab was missing. I updated the list of contributors.
I think that we can add a comment about that in the CONTRIBUTING.md file to avoid any future misunderstandings. what do you think?